src/components/NcAppSidebar/NcAppSidebarTabs.vue

```diff
@@ -273,7 +273,7 @@ export default {
 				border-radius: var(--default-grid-baseline) var(--default-grid-baseline) 0 0 !important;
 				margin: 0 !important;
 				border-bottom: var(--default-grid-baseline) solid transparent !important;
-				.checkbox-content__icon--checked > * {
+				.checkbox-content__icon > * {
```

Comment:
If the specificity is the same why not
`.checkbox-content__icon.checkbox-content__icon--checked` ?

Reply:
`--checked` covers only active tab, we need both